tests/browser.js

```diff
@@ -245,6 +246,17 @@ describe('Routing', function() {
       });
     });
 
+     it('navigates to external Urls', function(done) {
+      assertRendered('mainpage');
+      clickOn(app.refs.externalUrl);
+      delay(function() {
+        // Not sure how to vefify this case and continue.
+        // Browser gets redirected to external.
```

Comment:
Hm, I found a similar test below and arrived at this working test. But it isn't doing much.......

     it('navigates to external Urls', function(done) {
      assertRendered('mainpage');
      app.setProps({
        onClick: function(event) {
          // Make sure that the event hasn't had its default prevented by the
          // CaptureClicks component.
          assert(!event.defaultPrevented);
          event.preventDefault();
          done();
        }
      });
      clickOn(app.refs.externalUrl);
    });